src/main/java/com/ryanharter/autoparcel/AutoParcelExtension.java

```diff
@@ -85,7 +86,25 @@ String getParcelableCode(Context context, String className, String classToExtend
     return javaFile.toString();
   }
 
-  private Map<String, TypeName> convertPropertiesToTypes(Map<String, ExecutableElement> properties) {
+  private void validateProperties(ProcessingEnvironment env,
+                                  Map<String, ExecutableElement> properties) {
+    TypeElement parcelable = env.getElementUtils().getTypeElement("android.os.Parcelable");
```

Comment:
Just call `.asType()` here for parity with `serializable` variable and not having to do it inside the loop